src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -20,25 +21,28 @@
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
     private final AddressBook addressBook;
+    private final Countdown countdown;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
 
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
```

Comment:
I believe this comment needs a update since a new variable has been added to the constructor